server/src/main/java/org/opensearch/cluster/decommission/DecommissionController.java

```diff
@@ -267,4 +277,128 @@ public void clusterStateProcessed(String source, ClusterState oldState, ClusterS
             }
         });
     }
+
+    void setWeightForDecommissionedZone(List<String> zones, ActionListener<ClusterPutWRRWeightsResponse> listener) {
+        ClusterState clusterState = clusterService.getClusterApplierService().state();
+
+        DecommissionAttributeMetadata decommissionAttributeMetadata = clusterState.metadata().custom(DecommissionAttributeMetadata.TYPE);
```

Comment:
Use utility function created in metadata for getting decommission attribute - `clusterState.metadata().decommissionedAttribute()`

Reply:
Done. Used clusterState.metadata().decommissionedAttributeMetadata().